include/deal.II/grid/tria_objects.h

```diff
@@ -74,11 +62,56 @@ namespace internal
        */
       TriaObjects();
 
+      /**
+       * Constructor resetting some data.
```

Comment:
```suggestion
       * Constructor for a specific dimension.
```
Please also adjust the comment to the line above.